archlinuxcn/python-gluoncv-torch-git/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer: Butui Hu <hot123tea123@gmail.com>
+
+pkgname=python-gluoncv-torch-git
+_pkgver=0.0.3
+pkgver=0.0.3.r4.7759620
```

Comment:
It seems no available tag on GitHub repo. Why is it `0.0.3` tagged?

Reply:
Yeah, it's available from PyPI in version 0.0.3.